mealpy/mealpy.py

```diff
@@ -28,28 +29,30 @@
 
 CONFIG_FILENAME = 'config.yaml'
 COOKIES_FILENAME = 'cookies.txt'
+ROOT_DIR = Path(__file__).parent.parent
```

Comment:
This isn't getting the real root directory for me for some reason, maybe you need to do `.resolve()` before `.parent`?

Reply:
Can you explain the bug further and add a repro? It's working correctly for me.

```python
In [1]: template_config_path = ROOT_DIR / 'config.template.yaml'

In [2]: template_config_path.resolve()
Out[2]: PosixPath('/Users/ipwnponies/repos/mealpy/config.template.yaml')

In [3]: template_config_path.read_text()
Out[3]: "---\nemail_address: 'example@example.com'\nuse_keyring: False\n"
```

`resolve()` gets the real path (i.e. `readlink`) but that's not relevant to us, that's an OS-level detail.

Reply:
Oh, I found out I was running `python mealpy.py` within the inner directory instead of the root directory using `python mealpy/mealpy.py`, which might explain why the current code doesn't work for you. Switched to the second command and it works for me now.

Reply:
No, that still doesn't explain it. `__file__` is the module and not relative to `$CWD`. In fact, the snippet I pasted was ran from my `repos/` dir (i was testing if CWD affected the code) and it still works.

But as long as there's no issue, it's all good.

Reply:
This is what I get when I run `python mealpy.py` in the inner directory. I think if you run it from an outer directory the path will still work, because it will just have a longer path from the CWD to that file, but not an inner directory.

```
(Pdb) ROOT_DIR
PosixPath('.')
(Pdb) ROOT_DIR.resolve()
PosixPath('/Users/edmundmok/Desktop/repos/mealpy/mealpy')
(Pdb) ROOT_DIR.resolve().parent
PosixPath('/Users/edmundmok/Desktop/repos/mealpy')
(Pdb) template_config_path
PosixPath('config.template.yaml')
(Pdb) template_config_path.resolve()
PosixPath('/Users/edmundmok/Desktop/repos/mealpy/mealpy/config.template.yaml')
```

Reply:
Ohhhh.... That's dumb.
```python
In [20]: Path('.').parent
Out[20]: PosixPath('.')

In [21]: Path('.') == Path('.').parent.parent
Out[21]: True
```

The parent of `'.'` isn't `'..'`, as I intuitively though it would be. It's capped. Okay, I'll change to using resolved path.